pipeline/alignments/tokenizer.py

```diff
@@ -128,7 +128,7 @@ def _read_file_in_chunks(file_path, chunk_size):
             lines = file.readlines(chunk_size)
             if not lines:
                 break
-            yield lines
+            yield [line.rstrip() for line in lines]
```

Comment:
Is this just a newline? If so it would probably be better to do `[:-1]`.

Reply:
Ideally yes. The context here is that `\n` is already ignored in the ICU tokenizer but without this fix, in my production run I got one side of the corpus slightly longer than the other one (something like zh had ~100k lines more than en or vice versa) which is super weird. It seems something else except the `\n` causes line breaks (can it be `\r`?). I couldn't figure out what it was and how to deal with it, so stripping everything worked fine.